angular/src/directives/navigation/router-link-delegate.ts

```diff
@@ -52,6 +54,16 @@ export class RouterLinkDelegateDirective implements OnInit, OnChanges, OnDestroy
    */
   @HostListener('click', ['$event'])
   onClick(ev: UIEvent): void {
+    if (!this.elementRef.nativeElement.closest('ion-button')) {
+      // In case of CTRL+Click or Meta+Click,
+      // ignore those to allow default browser actions.
+      if (this.platform.is('desktop') && ev instanceof MouseEvent) {
```

Comment:
Why do we need to check if the device is on desktop/a MouseEvent?

Reply:
Agreed this appears to be redundant. The native `click` event is a [`MouseEvent`](https://developer.mozilla.org/en-US/docs/Web/API/Element/click_event). I'd advocate we remove the platform check to additionally handle tablet with a connected keyboard & mouse.

Reply:
Oftentimes, I tend to be too defensive with all the checks.

Will remove the if-statement.

Reply:
@Juarrow let me know when you have a moment to update the PR with this change.

Reply:
@sean-perkins just did. Also updated the commit message scope to reflect your change to the PR title.

Reply:
@sean-perkins @liamdebeasi anything happening?

Reply:
@Juarrow this task is in my sprint to both finish out and address this PR. The main issue effects each framework output. After looking deeper at `ion-button`, I would like to address to inconsistencies with that component (no action required on your end). 

Here's my WIP branch: https://github.com/ionic-team/ionic-framework/tree/FW-432 that pulls in your Angular change and applies a similar resolution to React and Vue. 

Once I am able to diagnose the `ion-button` behavior fully and add respective tests, I will likely open a new PR and give you co-author credit on the PR.